content/en/docs/concepts/services-networking/_index.md

```diff
@@ -1,63 +1,175 @@
+{{< /note >}}
+
+The network model describes how pods and their associated pod IPs can integrate with the larger network to support
+container networking.
+
+[comment]: <> (All diagrams.net figures are available at: https://drive.google.com/drive/folders/1MPOeuJ3wTzptutZX_6GKpLK8ljnojKE8?usp=sharing)
+
+[comment]: <> (good talk on K8 network models at https://www.cncf.io/wp-content/uploads/2020/08/CNCF_Webinar_-Kubernetes_network_models.pdf)
+
+Kubernetes IP addresses exist at the Pod scope. For example, on Linux, containers
+within a Pod share their network namespaces - including their IP address, and any
+network address from a lower layer, such as a MAC address.
+This means that containers within a Pod can all reach each other's ports on
+`localhost`. This also means that containers within a Pod must coordinate port
+usage (the same way that different processes on a physical server need to coordinate
+port use. This model, as used in Kubernetes, is called the _IP-per-pod_ model.
 
 How this is implemented is a detail of the particular container runtime in use.
 
-It is possible to request ports on the `Node` itself which forward to your `Pod`
-(called host ports), but this is a very niche operation. How that forwarding is
-implemented is also a detail of the container runtime. The `Pod` itself is
-blind to the existence or non-existence of host ports.
+It is possible to request and configure ports on the node itself (named _host ports_),
+that forward to a port on your Pod; however, this is a very niche operation.
+How that forwarding is implemented is also a detail of the container runtime.
+The Pod itself is not aware of the existence or non-existence of host ports.
 
-Kubernetes networking addresses four concerns:
-- Containers within a Pod [use networking to communicate](/docs/concepts/services-networking/dns-pod-service/) via loopback.
-- Cluster networking provides communication between different Pods.
-- The [Service](/docs/concepts/services-networking/service/) API lets you
-  [expose an application running in Pods](/docs/tutorials/services/connect-applications-service/)
-  to be reachable from outside your cluster.
-  - [Ingress](/docs/concepts/services-networking/ingress/) provides extra functionality
-    specifically for exposing HTTP applications, websites and APIs.
-- You can also use Services to
-  [publish services only for consumption inside your cluster](/docs/concepts/services-networking/service-traffic-policy/).
+## Network plugins
+
+[Network plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) set up IP
+addressing for Pods and their containers, and allow pods to communicate even when the source Pod and
+destination Pod are running on different nodes. Different network plugins achieve this in different ways
+with examples including tunneling or IP routing.
+
+{{< note >}}
+Network plugins are also known as _CNI_ or _CNI plugins_.
+{{< /note >}}
```

Comment:
What other kind of network plugins can I use with Kubernetes?

Reply:
All network plugins use CNI, but they don't _just_ use CNI.

People understand you're talking about network plugins when you say "CNI plugins" because CNI isn't used for anything _except_ network plugins. But these days, most of what a network plugin does doesn't involve CNI.

But anyway, doesn't need to be fixed in this PR.